modules/swagger-codegen/src/main/resources/JavaPlayFramework/routes.mustache

```diff
@@ -3,19 +3,19 @@
 # ~~~~
 
 {{#useSwaggerUI}}
-GET     /api                        controllers.ApiDocController.api
+    GET     /api                        controllers.ApiDocController.api
 {{/useSwaggerUI}}
 
 {{#apiInfo}}
-{{#apis}}
+    {{#apis}}
 
-#Functions for {{{baseName}}} API
-{{#operations}}
-{{#operation}}
-{{httpMethod}}     {{{contextPath}}}{{{path}}}                     controllers.{{classname}}Controller.{{operationId}}({{#pathParams}}{{paramName}}: {{{dataType}}}{{#hasMore}}, {{/hasMore}}{{/pathParams}})
-{{/operation}}
-{{/operations}}
-{{/apis}}
+        #Functions for {{{baseName}}} API
+        {{#operations}}
+            {{#operation}}
+                {{httpMethod}}     {{{contextPath}}}{{{path}}}                     controllers.{{classname}}Controller.{{operationId}}({{#pathParams}}{{paramName}}: {{#isUuid}}java.util.UUID{{/isUuid}}{{^isUuid}}{{{dataType}}}{{/isUuid}}{{#hasMore}}, {{/hasMore}}{{/pathParams}})
```

Comment:
I wonder if this should be done in the post processing in JavaPlayFrameworkCodegen.java instead. I think the dataType should always contain everything needed.

Reply:
@sparta15 have you tried the `fullJavaUtil` option to see if it would meet your requirement?
https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractJavaCodegen.java#L338

Reply:
Sorry, I have a lot of work and I couldn't review the comments. I'll try the `fullJavaUtil`option too